README.md

```diff
@@ -87,7 +87,11 @@ Getting started
    are not tracked by Git (optional):
 
     ```
-    coverage
+    echo "coverage" >> .gitignore
+    ```
+    Or if you use Windows for some reason:
```

Comment:
I think we should get rid of the `for some reason` part - no need to belittle Windows users.

Reply:
@PragTob Proudly speaking, never met ruby developer using Windows. So, imo, it will not touch anybody.

Reply:
I know quite some of them also including popular Rubyists :)

Reply:
@PragTob Got it, I add some row for this fanatics also :)